### PR TITLE
Code Review Request

### PR DESCRIPTION
**Pull request by dxania**
The project revolves around creating a clone page of [this](https://www.nytimes.com/2014/03/18/science/space/detection-of-waves-in-space-buttresses-landmark-theory-of-big-bang.html?_r=0) new york times article using floating & positioning. Main content of the page

- A navigation bar
- Article title & author
- The article content : text, images and a video
- Links to other articles (most popular, editors picks, more in space and cosmos)
- A side bar for signing up and links to the editors picks
- A footer with important links provided by new york times
- A sticky footer with account creation links

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,891 @@
...
+              <img src="images/art1-row1.jpg" alt="Article thumbnail" />
+              <figcaption>
+                Image by Space Telescope Science Institute/NASA, via
+                Associated Press
+              </figcaption>
+            </figure>
+            <span class="suggested-article-title">The Interstellar Comet Has Arrived in Time for the
+              Holidays</span>
+            <span class="suggested-article-date">Dec. 7</span>
+          </div>
```
Comment (pelzolga123) [inline index.html]: Each `div` in 'More in Space & Cosmos' section  should contain link to other article.

File: index.html [hunk 2]
```diff
@@ -0,0 +1,891 @@
...
+        <div class="suggested-article first-row-element">
+          <figure>
+            <img src="images/ep-1.jpg" alt="Article thumbnail" />
+            <figcaption>
+              Rhona Wise/EPA, via Shutterstock
+            </figcaption>
+          </figure>
+          <span class="suggested-article-title">A (Grudging) Defense of the $120,000 Banana</span>
+          <span class="suggested-article-date">Dec. 7</span>
+        </div>
```
Comment (pelzolga123) [inline index.html]: Each div in 'Editor's Picks' section should contain link to other article.

**Review by pelzolga123 — CHANGES_REQUESTED** [review]
Bottom part should contain links in leftmost part, pull request should be more descriptive.

**Comment by pelzolga123** [comment]
## Hi Team!  
### Your project looks amazing :+1: 

Next changes are required before project can be approved:
* [x] You have some text in your pull request description section, but it is not the right description.  Correct pull request description should have list of things you implemented in this project.
Look at [this link](https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/66243642/) from pathwright for detailed description.

* [x] Bottom part should contain links to other articles on the leftmost side. [Link to original website](https://www.nytimes.com/2014/03/18/science/space/detection-of-waves-in-space-buttresses-landmark-theory-of-big-bang.html?_r=0)

### Please kindly fix the above issues and send another code review request.

If you have any questions or need help, you can message me on slack (@ Olga Pelts). *Happy coding*

**Comment by dxania** [comment]
> Bottom part should contain links in leftmost part, pull request should be more descriptive.

Hey @pelzolga123 This is not really clear. Kindly elaborate or share a screenshot of this issue

**Comment by pelzolga123** [comment]
1. Bottom part should contain links to other articles on the leftmost side. That means each div in  'Editor's Picks' and  'More in Space & Cosmos'  sections should lead to an article. You can take look how it was done on original website and add the same links. I was talking about this sections:
image
image


2. Meaningful pull request name For example: New York Times clone 
Meaningful pull request description should contain information about things you added in this project. 
For example:

* Created fixed header and footer.
* Added embedded images and video.

And so on... It depends on you, how much to write, but it will be useful for people, who will look at your project in future. Because of this description project looks more professional. And for projects, when you will have 7-8 pull requests it will help to understand how your workflow.

**Comment by dxania** [comment]
> 1. Bottom part should contain links to other articles on the leftmost side. That means each div in  'Editor's Picks' and  'More in Space & Cosmos'  sections should lead to an article. You can take look how it was done on original website and add the same links. I was talking about this sections:
>    image
>    image
> 2. Meaningful pull request name For example: New York Times clone
>    Meaningful pull request description should contain information about things you added in this project.
>    For example:
> 
> * Created fixed header and footer.
> * Added embedded images and video.
>   And so on... It depends on you, how much to write, but it will be useful for people, who will look at your project in future. Because of this description project looks more professional. And for projects, when you will have 7-8 pull requests it will help to understand how your workflow.

Tha

> 1. Bottom part should contain links to other articles on the leftmost side. That means each div in  'Editor's Picks' and  'More in Space & Cosmos'  sections should lead to an article. You can take look how it was done on original website and add the same links. I was talking about this sections:
>    image
>    image
> 2. Meaningful pull request name For example: New York Times clone
>    Meaningful pull request description should contain information about things you added in this project.
>    For example:
> 
> * Created fixed header and footer.
> * Added embedded images and video.
>   And so on... It depends on you, how much to write, but it will be useful for people, who will look at your project in future. Because of this description project looks more professional. And for projects, when you will have 7-8 pull requests it will help to understand how your workflow.

Thanks. Much clearer

**Comment by shshamim63** [comment]
## Status :tada: [!! Approved !!] :tada: 
___
Hi @dxania  and @ivanid22 :wave:,
Well done :clap: , You have covered all the changes asked by the previous reviewer. Code looks neat and clean, no commented code, PR contains meaningful summary, readme contains info to give an overall idea about the project. No HTML markup validation error except some warning. In one word awesome. :sunglasses: 
___
#### This :+1:  is from me. You can merge it and go ahead.
___
## :herb: Happy Coding :herb: 
### Shakhawat Hossain
[Linkedin](https://www.linkedin.com/in/shakhawathossainshamim/) | [Slack](__Shakhawat Hossain__)

**Pull request merged**